src/styles/index.scss

```diff
@@ -1,3 +1,44 @@
+@import 'variables';
+@import 'stars';
+@import 'header';
+@import 'catalog.place';
+@import 'card';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.button {
```

Comment:
let's also move it to a separate file (or make it an element of card) - it's a separate BEM block